### PR TITLE
Update readme by writing about 3_Inference IPython Notebook 

### PR DESCRIPTION
**Pull request by tarunjain1st**
done with issue #11

### CODE REVIEW

**Review by srimanikantapalakollu — APPROVED** [review]
Soon, we are going to merge it.

File: project_2_image_captioning_project/README.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ The decoder model used is the same one from [Show, Attend and Tell: Neural Image
 
    The optimizer used is Adam optimizer. We conclude with the training notebook here and go to the next phase.
 
+- **3_Inference:** The purpose of this file is to make the predictions by loading `trained model` and `vocabulary file` to get the desired result. This model generates good captions for the provided image but it can always be improved later by including hyper-parameters and using more accurate algorithms.
```
Comment (vidhi-mody) [inline project_2_image_captioning_project/README.md]: Can you replace the title with 3_Inference.ipynb to maintain consistency?

**Review by vidhi-mody — CHANGES_REQUESTED** [review]

File: project_2_image_captioning_project/README.md (hunk 1 above)
Comment (vidhi-mody) [reply project_2_image_captioning_project/README.md]: @tarunjain1st can you please make this minor change so that I can merge this PR?

Comment (tarunjain1st) [reply project_2_image_captioning_project/README.md]: yeah sure

**Comment by tarunjain1st** [comment]
made changes

**Review by vidhi-mody — APPROVED** [review]

**Pull request merged**